backend/src/main/java/ca/bc/gov/hlth/mohums/exceptions/ErrorResponse.java

```diff
@@ -0,0 +1,51 @@
+package ca.bc.gov.hlth.mohums.exceptions;
+
+import java.time.LocalDateTime;
+
+public class ErrorResponse {
+    private LocalDateTime timestamp;
+    private int status;
+    private String error;
+    private String message;
+    private String path;
```

Comment:
In addition to the above, I'm wondering what inspired this structure. It's good, but ironically that's part of what makes it look odd to me. It's used only by bulk upload, so in some sense it's a bulk upload error response, but the name and structure is generic and seems to imply requirements we don't actually have, like the need to set a path and timestamp.

There is some maintenance risk in offering a schema before we're prepared to support it. Because UMC is the only approved caller of UMS for bulk removal, we don't have much to worry about, but it's still something to consider.

One way to "future proof" it a bit is to follow existing standards. For that, you could consider [RFC 9457](https://datatracker.ietf.org/doc/html/rfc9457). It's a bit much to call it an "existing standard," as it was only published in July, and it's arguably not even a necessary standard, but if nothing else, standardization helps reduce the need for conversation on low-value topics.

If you did use this standard, `ErrorResponse` would be refactored like this:

**BulkRemovalProblemDetails Class (following RFC 9457)**
```java
package ca.bc.gov.hlth.mohums.exceptions;

import java.net.URI;

public class BulkRemovalProblemDetails {
    private URI type;
    private String title;
    private int status;
    private String detail;
    private URI instance;

    public BulkRemovalProblemDetails() {
        this.type = URI.create("BulkRemovalProblemDetails");
        this.title = "Bulk Removal Error";
        this.status = 400;
        this.detail = "";
        this.instance = URI.create("about:blank");
    }

    // Getters and Setters

}
```

**BulkRemovalExceptionHandler Class**
```java
package ca.bc.gov.hlth.mohums.exceptions;

import org.springframework.http.HttpStatus;
import org.springframework.http.ResponseEntity;
import org.springframework.web.bind.annotation.ControllerAdvice;
import org.springframework.web.bind.annotation.ExceptionHandler;
import org.springframework.web.context.request.WebRequest;

@ControllerAdvice
public class BulkRemovalExceptionHandler {

    @ExceptionHandler(BulkRemovalRequestException.class)
    public ResponseEntity<BulkRemovalProblemDetails> handleBulkRemovalExceptions(BulkRemovalRequestException ex, WebRequest request) {
        BulkRemovalProblemDetails problemDetails = new BulkRemovalProblemDetails();
        problemDetails.setDetail(ex.getMessage());

        return new ResponseEntity<>(problemDetails, HttpStatus.BAD_REQUEST);
    }
}
```

This approach ensures that our error responses are consistent, standardized, and easier to maintain in the long run.

THAT SAID, I only learned about RFC 9457 today, so I'm not claiming it's a problem-free improvement, just an illustrative example.